### PR TITLE
Use libcalico-go v1.0.0-rc4

### PR DESCRIPTION
**Pull request by neiljerram**

### CODE REVIEW

**Comment by neiljerram** [comment]
retest this please

File: utils/gen-version.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ version_file=python/version.py
 
 # Get the location of this script.  Other scripts that we use must be in the
 # same location.
-scriptdir=$(dirname $(readlink -f $0))
+scriptdir=$(dirname $(realpath $0))
```
Comment (fasaxc) [inline utils/gen-version.sh]: Why this change?  Googling suggests `realpath` is being deprecated in favour of `readlink -f`

**Comment by neiljerram** [comment]
It's because Casey was doing part of the release process, late last night (after my home laptop was failing bizarrely), and his Mac doesn't have readlink -f.Although after that change, it turned out the Mac didn't have ldd either, so still couldn't build the artifacts. I agree we should probably revert this change, or maybe simplify to just `dirname $0`. I think we should also automate further so that artifact building is done on a known machine.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         From: Shaun CramptonSent: Wednesday, 7 December 2016 08:54To: projectcalico/felixReply To: projectcalico/felixCc: Neil Jerram; AuthorSubject: Re: [projectcalico/felix] Use libcalico-go v1.0.0-rc4 (#1215)@fasaxc commented on this pull request.



In utils/gen-version.sh:
> @@ -8,7 +8,7 @@ version_file=python/version.py
 
 # Get the location of this script.  Other scripts that we use must be in the
 # same location.
-scriptdir=$(dirname $(readlink -f $0))
+scriptdir=$(dirname $(realpath $0))

Why this change?  Googling suggests realpath is being deprecated in favour of readlink -f

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/projectcalico/felix","title":"projectcalico/felix","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/projectcalico/felix"}},"updates":{"snippets":[{"icon":"PERSON","message":"@fasaxc commented on #1215"}],"action":{"name":"View Pull Request","url":"https://github.com/projectcalico/felix/pull/1215#pullrequestreview-11756419"}}}
